src/Menu.php

```diff
@@ -356,7 +356,9 @@ private function renderItems(array $items): string
                     $lines[] = $item['label'];
                 }
             } else {
-                $lines[] = Html::tag($tag, $menu, $options);
+                $lines[] = $tag === false
```

Comment:
Was it previously accepting `false`?

Reply:
Yes. Previous `Html::tag(false, $content)` returned pure `$content` without tag.